app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC

```

Comment:
Player must be abstract, not a dwarf

Reply:
![image](https://github.com/user-attachments/assets/4ce8bcb5-b240-45bb-b01b-569984080cbb)
writes that I need to use an abstract